docs/static/running-logstash-command-line.asciidoc

```diff
@@ -153,6 +153,11 @@ With this command, Logstash concatenates three config files, `/tmp/one`, `/tmp/t
   When creating pipeline batches, how long to wait while polling for the next event. This option defines
   how long in milliseconds to wait while polling for the next event before dispatching an undersized batch
   to filters and outputs. The default is 50ms.
+  
+  *`--pipeline.ecs_compatibility MODE`*::
+  Sets the ECS compatibility mode for the whole Logstash process, see <<ecs-compatibility>>.
+  Can be an ECS version like `v8` or `disabled`.
+  The default depends on the individual plugins.
```

Comment:
I'm adapting the text we have for the [`pipeline.ecs_compatibility`](https://github.com/elastic/logstash/blob/70fb72b584d08600d4cb1d08c6f4e31e9b572a07/config/logstash.yml#L69-L81) config file option.

```suggestion
  Sets the process default value for  ECS compatibility mode.
  Can be an ECS version like `v1` or `v8`, or `disabled`.
  The default is `v8`.
  Pipelines defined before Logstash 8 operated without ECS in mind.
  To ensure a migrated pipeline continues to operate as it did in older releases of Logstash, opt-OUT of ECS for the individual pipeline by setting `pipeline.ecs_compatibility: disabled` in its `pipelines.yml` definition.
  Using the command-line flag will set the default for _all_ pipelines, including new ones.
  See <<ecs-compatibility>> for more info.
```